components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -41,9 +41,9 @@ export class ServicesSidebarComponent implements OnInit {
       (state) => state.servicesHealthSummary));
 
     this.currentPage = 1;
-
-    // this.servicesHealthSummary$.subscribe(healthSummary =>
-    // this.totalServices = healthSummary.total);
+    this.servicesHealthSummary$.subscribe((healthSummary) => {
+      this.totalServices = healthSummary.total;
```

Comment:
If this field may or may not exist I would probably use `get()` from lodash/fp to get the value. It will always return a value even if one of the parent objects in undefined.

It should look something like `this.totalServices = get('total', healthSummary)`

This is the documentation from lodash, https://lodash.com/docs/4.17.11#get, but also take a look at the fp docs to see how to convert the syntax. https://github.com/lodash/lodash/wiki/FP-Guide

Reply:
I can do that! Thanks for your feedback Matt.